src/renderer/nav/Validator.tsx

```diff
@@ -37,9 +39,25 @@ const Validator = () => {
   const validator = useAppSelector(selectValidatorNetworkState);
   const validatorImageName = 'cryptoworkbench/solana-amman';
   const [validatorImageTag, setValidatorImageTag] = useState<string>('');
-  const [validatorImageTags, setValidatorImageTags] = useState<string[]>([]);
+  // const [validatorImageTags, setValidatorImageTags] = useState<string[]>([]);
   const [containerInspect, setContainerInspect] = useState<any>({});
 
+  const {
+    /* isLoading: validatorImageTagsIsLoading, */
+    /* error: validatorImageTagsError, */
+    data: validatorImageTagsData,
+  } = useQuery('validatorImageTags', () =>
+    window.promiseIpc.send('DOCKER-GetImageTags', validatorImageName)
+  );
+
+  const validatorImageTags = validatorImageTagsData || [
+    'Image list loading....',
+  ];
+
+  // TODO: not sure how to tell the user if we fail to get the list of image tags...
```

Comment:
Maybe you could make the embedded bit JSX instead of a string and use error text? FWIW, you can do stuff like:

```typescript
const tagDropdownDisplay = (
  <div className="text-red-500">
    An error occurred getting image tags.
  </div>
);

return (
  <div>{tagDropdownDisplay}</div>
);
```